timescale-forge/disk-management.md

```diff
@@ -0,0 +1,89 @@
+# Disk management
+
+Timescale Forge contains several mechanisms for managing disk space on your
+services. There are four key tasks that Forge performs to handle disk space:
+
+1.  Detect if storage capacity begins to fill up
+1.  Notify you about the growth
+1.  Automatically activate overload protections
+1.  Allow you to return your database to a normal state
+
+This section explains what the various mechanisms are, and how to best make use
+of them.
+
+## Continuous storage monitoring
+
+Timescale Forge continuously monitors the health and resource consumption of all
+database services. You can check your health data by navigating to the `metrics`
+tab in your service dashboard. These metrics are also monitored by the Timescale
+operations team.
+
+*Insert screenshot here*
+
+If your database exceeds a storage threshold of available resources, some
+automated actions are triggered, including notifications, and preventative
+actions.
+
+## Automated user alerting
+
+When your disk usage exceeds certain thresholds, you will recieve email notifications. These notifications occur at:
+
+*   75%
+*   85%
+*   95%
+
+So that you aren't overwhelmed by automated message, the alerting thresholds use low- and high-watermarks, and only one message is sent per database per day.
+
+## Automated overload protection
+
+If your database continues to increase in size past these thresholds, automated overload protection is activated when your disk becomes 99% full. When this happens, your database is put into read-only mode, and you receive a notification on email and in the Timescale Forge console.
```

Comment:
The Forge console won't give a notification, but the change in status is visible so maybe something like:
```suggestion
If your database continues to increase in size past these thresholds, automated overload protection is activated when your disk becomes 99% full. When this happens, your database is put into read-only mode, you receive a notification email and the changed status is visible in the Timescale Forge console.
```
or similar

Reply:
```suggestion
If your database continues to increase in size past these thresholds, automated overload protection is activated when your disk becomes 99% full. When this happens, your database is put into read-only mode, and you receive a notification on email and the Timescale Forge console shows the changed status.
```
??